### PR TITLE
chore: Update GitHub workflows

### PR DESCRIPTION
**Pull request by fussel178**
### Summary 

Update of GitHub Action workflows

### Details 

I've updated the workflow dependencies and cleaned up some inconsistent and missing parts.

### CLA

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

### CODE REVIEW

**Review by jvpichowski — APPROVED** [review]
LGTM 😄

**Comment by fussel178** [comment]
Also implement https://github.com/wuespace/telestion-core/issues/258

File: project.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "version": "0.0.19"
```
Comment (jvpichowski) [inline project.json:2]: I like to revert to v0.0.1.

**Review by jvpichowski — CHANGES_REQUESTED** [review]
Thanks but could you change the initial version?

File: project.json (hunk 1 above)
Comment (fussel178) [reply project.json:2]: Of course!

**Comment by jvpichowski** [comment]
Is it possible to update the version in the build files too, before publishing?  Like here: https://github.com/wuespace/telestion-core/blob/ee6991db241feb30d842bea5b6da05e598fe21da/modules/telestion-api/build.gradle#L30

I can make a central variable, if it helps.

File: project.json (hunk 1 above)
Comment (fussel178) [reply project.json]: The Conventional Release Action automatically determines the next version number for you, so I resetted it to `v0.0.0`.

**Review by jvpichowski — APPROVED** [review]

**Review by jvpichowski — APPROVED** [review]

**Pull request merged**